### ISSUE TITLE
resCode 5031 kills EV6 battery?

### ISSUE THREAD

**Issue by Henry1693** [SondreNjaastad/node-red-contrib-bluelinky#95]
I'm polling my Kia EV6 every 30 minutes using 'Get full status'. This week the service gave an error
`retCode:F,resCode:5031,resMsg:Unavailable remote control - Service Temporary Unavailable,msgId:914acf90-a07d-11ef-8d98-adc9117f4e8a`
and after that every next call gave
`retCode:F,resCode:4004,resMsg:Duplicate request - Duplicate request,msgId:e3702e60-a081-11ef-82af-55cf2d4005a9`
Maybe a coincidence but when I tried to start the car in the morning the 12 volt battery was dead.
This happend twice in a row this week, so I stopped the Node-RED flow to see if the battery stays alive, and it did.
Could it be that because of the number of retries the battery gets deflated?

**Comment by PierreLevres** [comment]
Yes. Pulling data from the car with “refresh” drains the battery. It is because several systems stay active and do not get into sleep/suspend mode.
________________________________
Van: Henry1693 ***@***.***>
Verzonden: Thursday, November 14, 2024 9:57:12 AM
Aan: SondreNjaastad/node-red-contrib-bluelinky ***@***.***>
CC: Subscribed ***@***.***>
Onderwerp: [SondreNjaastad/node-red-contrib-bluelinky] resCode 5031 kills EV6 battery? (Issue #95)


I'm polling my Kia EV6 every 30 minutes using 'Get full status'. This week the service gave an error
retCode:F,resCode:5031,resMsg:Unavailable remote control - Service Temporary Unavailable,msgId:914acf90-a07d-11ef-8d98-adc9117f4e8a
and after that every next call gave
retCode:F,resCode:4004,resMsg:Duplicate request - Duplicate request,msgId:e3702e60-a081-11ef-82af-55cf2d4005a9
Maybe a coincidence but when I tried to start the car in the morning the 12 volt battery was dead.
This happend twice in a row this week, so I stopped the Node-RED flow to see if the battery stays alive, and it did.
Could it be that because of the number of retries the battery gets deflated?

**Comment by Henry1693** [comment]
I understand that refreshing drains the battery, but I'm only polling the data every 30 minutes. As far as I know there is also a retry counter in the code?
When there is no error I can refresh every 10 minutes for 24 hours without any problems, so question is should I stop polling the car when I receive an error to prevent the draining of the battery?
